src/app.js

```diff
@@ -5,3 +5,21 @@ console.log(`Application: "${config.name}" has been started`);
 
 let user1 = new models.User();
 let product1 = new models.Product();
+let dirWatcher = new models.DirWatcher();
+let importer = models.Importer;
+
+dirWatcher.on('dirwatcher:changed', path => {
```

Comment:
As far as I can see, it does not react to changes in the directory (new files).

Reply:
It works for new files because in a method `setPathWatcher()` of a `DirWatcher` service we set watcher on a whole directory, so if something will be changed inside this directory, `dirWatcher` will emitted the `'dirwatcher:changed'` event.